sample.env.local

```diff
@@ -0,0 +1,7 @@
+# Rename this file to env.local and fill out the information below to get started
```

Comment:
Just a really tiny typo: file needs to be renamed to `.env.local` 🤓

Reply:
Good eyes, thx!